src/components/ServiceDetails/index.js

```diff
@@ -62,7 +71,7 @@ class ServiceDetails extends Component {
           star_rating={service.service_rating}
           totalRating={service.total_users_rated}
         />
-        <PricingDetails price_model={service.price_model} />
+        <PricingDetails price_model={service.price_model} price_strategy={service.pricing_strategy} />
```

Comment:
`pricing_model` could be inferred from `price_strategy`.

Reply:
Done